mmv1/third_party/terraform/transport/transport.go

```diff
@@ -189,3 +189,140 @@ func IsApiNotEnabledError(err error) bool {
 	}
 	return false
 }
+
+type GetPaginatedItemsSliceOptions struct {
```

Comment:
nit: Can you document the fields on this, given it's a public function's interface?